core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -234,6 +235,8 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
           val deserializationTimes = validTasks.map { case TaskUIData(_, metrics, _) =>
             metrics.get.executorDeserializeTime.toDouble
           }
+          graphData("Task Deserialization Time") = deserializationTimes.mkString(",")
```

Comment:
Rather than creating your own string representation of a list, can do the necessary conversation to pass proper JSON lists here?